spec/factories/nca_facilities.rb

```diff
@@ -121,6 +124,7 @@
     facility_type 'va_cemetery'
     classification 'Rural'
     website nil
+    mobile nil
```

Comment:
Why would all of these be nil? Isn't a cemetery clearly not mobile (`false`) or is this a limitation of the data?

Reply:
We don't have a data source to confirm this yet, which is why we're filling in nil for mobile in the other facility types. Although a mobile cemetery does seem highly unlikely...